lib/modules/dosomething/dosomething_reportback/includes/ReportbackItemTransformer.php

```diff
@@ -18,6 +18,7 @@ public function index($parameters) {
     }
     catch (Exception $error) {
       // @TODO: Potentially log error to watchdog.
+      http_response_code('404');
```

Comment:
@aaronschachter I forget why we'd want a 200 if there are no results found again?